src/lightning/pytorch/__init__.py

```diff
@@ -1,5 +1,7 @@
 """Root package info."""
 
+from __future__ import annotations
```

Comment:
We'd need to remove all these unused additions. Maybe we can write a script that processes the git diff and undoes it if this is the only change

Reply:
```bash
for f in $(git diff --name-only master..HEAD); do
  if [ "$(git diff master..HEAD $f | sed -n '/^+++/d; /^+$/d; /^+/p')" = "+from __future__ import annotations" ]; then
    git checkout master -- $f
  fi
done
```